libs/services/designer-client-services/src/lib/standard/manifests/xml.ts

```diff
@@ -0,0 +1,210 @@
+import { SettingScope } from '@microsoft/utils-logic-apps';
+import type { OperationManifest } from '@microsoft/utils-logic-apps';
+
+const connector: any = {
+  id: 'connectionProviders/xmlOperations',
+  name: 'xmlOperations',
+  properties: {
+    description: 'XML Operations',
+    displayName: 'XML Operations',
+  },
+};
+
+const settings: any = {
+  secureData: {},
+  trackedProperties: {
+    scopes: [SettingScope.Action],
+  },
+};
```

Comment:
Did we verify from consumption portal this action only has these two settings?

Reply:
As far as I can tell these settings match the current consumption portal settings